From: dju` <dju@elegiac.net>
Subject: fix mistake in Solid action predicate for kamera
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=585717

The predicate in file /usr/share/kde4/apps/solid/actions/solid_camera.desktop
is wrong:

X-KDE-Solid-Predicate=Camera.supportedDrivers & 'gphoto'

whereas it should be:

X-KDE-Solid-Predicate=Camera.supportedDrivers == 'gphoto'

With this correction the "Open with File Manager" action appears in the action
list when connecting a gphoto camera to the computer.


--- a/kamera/solid_camera.desktop
+++ b/kamera/solid_camera.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-X-KDE-Solid-Predicate=Camera.supportedDrivers & 'gphoto'
+X-KDE-Solid-Predicate=Camera.supportedDrivers == 'gphoto'
 Type=Service
 Actions=open;
 
